### ISSUE TITLE
"The ServiceAPI is currently offline"

### ISSUE THREAD

**Issue by flashkiller999** [s0faking/plugin.video.orftvthek#129]
Hello,

since a few days now the ServiceAPI is no longer working.
Switching to HTML parsing works, but does not offer the same feature set.

Maybe you could take a look at it, if I can provide specific logs please advice on how.
Simply enabling logging in Kodi seems to not provide more infos from the plugin:
```
2023-06-18 21:59:11.812 T:3670    DEBUG <CAddonSettings[plugin.video.orftvthek]>: trying to load setting definitions from old format...
2023-06-18 21:59:11.813 T:3670    DEBUG <CSettingsManager>: requested setting (videoDeliveryProgressive) was not found.
2023-06-18 21:59:11.813 T:3670    DEBUG <CSettingsManager>: requested setting (videoQuality) was not found.
2023-06-18 21:59:11.813 T:3670    DEBUG <general>: Service API activated
2023-06-18 21:59:11.813 T:3670    DEBUG <general>: ServiceAPI  - Init done
2023-06-18 21:59:12.264 T:3670     INFO <general>: CPythonInvoker(112, /storage/.kodi/addons/plugin.video.orftvthek/default.py): script successfully run
2023-06-18 21:59:12.265 T:3669    DEBUG <general>: Thread waiting 3184521856 terminating
2023-06-18 21:59:12.265 T:3670    DEBUG <general>: onExecutionDone(112, /storage/.kodi/addons/plugin.video.orftvthek/default.py)
2023-06-18 21:59:12.292 T:815     DEBUG <general>: ------ Window Deinit (DialogBusy.xml) ------
2023-06-18 21:59:12.297 T:3671    DEBUG <general>: Thread BackgroundLoader start, auto delete: false
2023-06-18 21:59:12.306 T:3671    DEBUG <general>: Thread BackgroundLoader 3457462912 terminating
2023-06-18 21:59:12.342 T:815     DEBUG <general>: ------ Window Init (DialogNotification.xml) ------
2023-06-18 21:59:12.668 T:3670     INFO <general>: Python interpreter stopped
2023-06-18 21:59:12.668 T:3670    DEBUG <general>: Thread LanguageInvoker 3296981632 terminating
2023-06-18 21:59:20.081 T:815     DEBUG <general>: ------ Window Deinit (DialogNotification.xml) ------
```

Thanks in advance for looking at it

**Comment by s0faking** [comment]
yep, the endpoint */livestreams/24hours?limit=20 returns error 500 atm. i will take a look. might be just some problem on the tvthek servers. doesn't look like the disabled the 24 hours view

**Comment by s0faking** [comment]
fixed with 0.12.9
